First pass at Variety 2020 and VIP

emeaguiar opened the pull request:
This is a first pass at the Variety 2020 and VIP tokens needed to build a few components in the site, mainly basic fonts, footer, and article header components.

laras126 commented on packages/larva-tokens/src/brands/variety.json:
```diff
@@ -0,0 +1,60 @@
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_BASIC": {
```
Anything that is a webfont needs to have a fallback - basic is intended to be a websafe font like Helvetica. If it is a webfont, it should be added as FONT_FAMILY_BASIC_FANCY with a fallback, FONT_FAMILY_BASIC.

laras126 replied:
Also - can this be FONT_FAMILY_MONO, as above? The tokens between Variety and Variety VIP should be as similar as possible

emeaguiar replied:
This is an overlook in my part `FONT_FAMILY_MONO` should be `FONT_FAMILY_ACCENT` for vip, which I'll update. Thanks for the catch!

These names are taken from the invision styleguides, where Variety and VIP share a naming convention, but differ in values.

laras126 commented on packages/larva-tokens/src/brands/variety.json:
```diff
@@ -0,0 +1,60 @@
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_ACCENT": {
```
Same note as below about websafe font version.

emeaguiar replied:
I've updated with fallbacks

laras126 commented on packages/larva-tokens/src/brands/variety-vip.json:
```diff
@@ -0,0 +1,60 @@
+			"type": "font-family",
+			"category": "font-family"
+		},
+		"FONT_FAMILY_MONO": {
```
What is the reason for having IBM_PLEX_MONO as FONT_FAMILY_MONO here vs. FONT_FAMILY_BASIC below? Can they be the same?

emeaguiar replied:
As I mentioned above, this should be `FONT_FAMILY_ACCENT` for vip and `FONT_FAMILY_BASIC` for variety based on the styleguides, I will update.

laras126 commented on packages/larva-tokens/build/variety.custom-properties.css:
```diff
@@ -0,0 +1,33 @@
+  --font-family-body-fancy: 'IBM Plex Serif', serif;
+  --border-color-brand-primary: rgb(65, 126, 244);
+  --background-color-brand-primary: rgb(238, 232, 9);
+  --background-color-brand-accent-a-100-b: rgb(26, 40, 47);
```
Where is this coming from? What does `a-100-b` represent?

For colors, I'm also guessing there will be several for VY that are not in the existing tokens. Added some guidelines for that here: https://github.com/penske-media-corp/pmc-larva/wiki/CSS-Class-Generators#basic-utility-generator

emeaguiar commented:
@laras126 I made some updates and remove the variants of the colors to put them in the theme itself, make sense to me since this styleguide has a lot of colors. So let me know if something else caught our eyes.

laras126 commented on packages/larva-tokens/src/brands/variety-vip.json:
```diff
@@ -0,0 +1,76 @@
+			"category": "font-family"
+		},
+		"FONT_FAMILY_SECONDARY": {
+			"value": "{!BONNIE}",
```
This should be a web safe font

laras126 commented:
@emeaguiar Almost there! 

I think the naming for the webfonts is confusing - `_FANCY` = webfont, (in retrospect _WEBFONT may be a better name...) and every font family should have a base version e.g. FONT_FAMILY_ACCENT set to a fallback font, and then the webfont assigned to `_FANCY`.

So, from what I see in the PR, there needs to be fallback for FONT_FAMILY_ACCENT, and it looks like a webfont is assigned to FONT_FAMILY_SECONDARY, when that should be the fallback font.

emeaguiar commented:
Hey @laras126 , thanks for the explanation!

I have updated the fallback fonts for the secondary typeface and added a new accent fallback, let me know if there's anything else that needs to be done.

emeaguiar commented:
Hey @laras126 just reminding you this is here, let me know if something else is needed

mehigh commented:
@laras126 can you review in order to plan to get it merged in prior to the holidays season? Anything we can support to get this moving?

mehigh commented:
@laras126 can we at least get a response on when we can expect this to get merged in?

laras126 commented:
@mehigh Looking now - sorry about the delay. Was OOO last week, and will merge this today.
